src/System.Net.WebSockets.Client/src/System.Net.WebSockets.Client.csproj

```diff
@@ -6,13 +6,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
-    <Configuration Condition=" '$(Configuration)' == '' ">Windows_Debug</Configuration>
```

Comment:
Does this still work since we don't have a "Debug" configuration for this project?